### PR TITLE
feat(textile): Replace Temporal with Textile

### PR DESCRIPTION
**Pull request by danielhumgon**
PR related to this trello task --> https://trello.com/c/RuDRG9R1

### CODE REVIEW

**Comment by danielhumgon** [comment]
This is no longer a Work in Progress, is ready for @christroutner to review

**Review by christroutner — APPROVED** [review]

**Comment by christroutner** [comment]
:tada: This PR is included in version 1.5.0 :tada:

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

File: src/lib/bch.js [hunk 1]
```diff
@@ -834,6 +742,41 @@ class BCH {
...
+      // Auth to hub
+      const { id } = await this.textile.authUser()
+      // Init bucket
+      const { buckets, bucketKey } = await this.textile.initBucket(id)
+
+      // Get file buffer
+      const buff = this.fs.createReadStream(relFilePath)
+
+      // Push file
+      const result = await textile.pushPath(buckets, bucketKey, buff, fileName)
```
Comment (rkalis) [inline src/lib/bch.js:767]: Should this be `this.textile.pushPath` for consistency?

Comment (christroutner) [reply src/lib/bch.js:767]: Thanks! Nice catch!

**Pull request merged**